### DISCUSSION TITLE
New Networking modes still require Insiders build?

### DISCUSSION THREAD

**Discussion by diehard2** [microsoft/WSL#10776] [General]
I see that the features were moved from [experimental] to [wsl2]. However, the blog announcing them says that it will only work with the latest insider build version of Windows. Is this still true now that its been moved from [experimental]?

**Comment by Ynewtime** [comment]
Looks like so. Still WSL 2.0.0+ is pre-release and not stable yet. Also I am using Windows 11 23H2 now but having the same issue (networkingMode=mirrored doesn't work) as 22H2.

**Reply by diehard2** [reply]
@Ynewtime 2.0.9 is a full fledged release

**Comment by J0F3** [comment]
Yea, I was wondering the same. I have got the update to WSL 2.0.9 on Wednesday and the new features like `networkingMode=mirrored` work even on 22H2 now. The documentation seems be outdated currently. 
I only noticed the update because I got a popup about proxy setting changes regarding to the `autoproxy` feature. Then I tried also the other settings and they just work even on 22H2 now.
Looks like there is a release announcement blog post or so missing. 😃
